.storybook/main.ts

```diff
@@ -19,9 +22,18 @@ module.exports = {
   },
   features: {
     storyStoreV7: true,
+    babelModeV7: true,
   },
   async viteFinal(config, { configType }) {
     config.base = process.env.BASE_PATH || config.base;
+    config.plugins = [
+      ...config.plugins,
+      svgr({
+        svgrOptions: {
+          template,
+        },
+      }),
+    ];
```

Comment:
My understanding is that these config options are telling storybook how to handle the svg components in this PR.

Thinking big picture about this being a design system that will be consumed by other applications, will other applications have to make similar config changes to run the icons, or is this something that only needs to be done for storybook?

If there are implications for consumer applications, it might be best to in the `README.md` to note prerequisites for running the toolkit (or specifically icons)

Reply:
This will bundle the generated react components into the build. They can be imported from the bundle like any other component.